bpf/http_ssl_defs.h

```diff
@@ -180,12 +180,6 @@ handle_ssl_buf(void *ctx, u64 id, ssl_args_t *args, int bytes_len, u8 direction)
             bpf_dbg_printk("SSL conn");
             dbg_print_http_connection_info(&conn->p_conn.conn);
 
-            // unsigned char buf[48];
```

Comment:
is this intended to be commented? or is a piece of code to debug things?

Reply:
Yeah I found some old debugging code and I cleaned it up. We don't need it really and if we ever did we can easily write something like that.